pkgs/development/compilers/llvm/12/llvm/default.nix

```
@@ -58,6 +58,9 @@ in stdenv.mkDerivation (rec {
     ++ [ zlib ];
 
   patches = [
+    # Fix llvm being miscompiled by some gccs. See llvm/llvm-project#49955
```

Rather avoid the github shorthand I cannot click on
```suggestion
    # Fix llvm being miscompiled by some gccs. See https://github.com/llvm/llvm-project/issues/49955
```